docs/using-the-compiler.rst

```diff
@@ -147,14 +147,14 @@ Target Options
 Below is a list of target EVM versions and the compiler-relevant changes introduced
 at each version. Backward compatibility is not guaranteed between each version.
 
-- ``homestead``
+- ``homestead`` (*support deprecated*)
```

Comment:
Support can't be deprecated, so I'd just change this to
```suggestion
- ``homestead`` (*deprecated*)
```

Reply:
I intentionally worded it that way. It's not the EVM version that we're deprecating - we have no control over that and I have no idea if these old EVMs are considered deprecated or not. AFAIK there's no deprecation procedure for hard forks. They just naturally fall out of use. This fuzziness about how relevant they're still are is exactly what has been holding us back from doing this deprecation earlier.

What we're doing is deprecating compiler's support for these EVM versions.

I agree that the wording is a bit awkward but I did not find a better way to word it concisely.

Reply:
Eh, alright, fair enough. Merge away.